pandas/tseries/tests/test_base.py

```diff
@@ -45,6 +45,14 @@ def test_ops_properties_basic(self):
         self.assertEqual(s.day,10)
         self.assertRaises(AttributeError, lambda : s.weekday)
 
+    def test_shift_index(self):
```

Comment:
move this to ``test_index.py`` inside that ``Base`` class

further you will need to create the index with ``self.create_index()``